src/components/MainLayout/MainHeader.tsx

```diff
@@ -314,20 +232,90 @@ export const HeaderNav = () => {
   });
 
   return (
-    <div className={classes.navLinks}>
+    <Box
+      css={{
+        justifySelf: 'stretch',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'flex-end',
+        '@sm': {
+          alignItems: 'flex-start',
+          flexDirection: 'column',
+        },
+      }}
+    >
       {navItems.map(navItem => (
-        <NavLink
-          className={classes.navLink}
-          isActive={(nothing, location) =>
+        <Link
+          css={{
+            my: 0,
+            mx: '$md',
+            fontSize: '$6',
+            fontWeight: '$bold',
+            color: '$white',
+            textDecoration: 'none',
+            py: '$xs1',
+            px: 0,
+            position: 'relative',
+            '&::after': {
+              content: `" "`,
+              position: 'absolute',
+              left: '50%',
+              right: '50%',
+              backgroundColor: '$mediumRed',
+              transition: 'all 0.3s',
+              bottom: 0,
+              height: '$2xs',
```

Comment:
you can remove this token from the theme and just hardcode this value since it's not a layout property
```suggestion
              height: '2px',
```